test/tools/runner/filters/client_encryption_filter.ts

```diff
@@ -24,15 +58,18 @@ export class ClientSideEncryptionFilter extends Filter {
   enabled: boolean;
   static version = null;
   static libmongocrypt: string | null = null;
+  static cryptShared: AutoEncrypter['cryptSharedLibVersionInfo'] | null = null;
```

Comment:
@durran FYI I pushed this change up after your initial review.  

I'm happy to revert, but I think this might give us some safety going forward.  By forcing the shared library to be loaded, if it is configured in `CRYPT_SHARED_LIB_PATH`, this will make any new tests we add fail if they either don't configure crypt_shared but they're supposed to or if they attempt to load crypt_shared from the wrong path.  (ex: `13. MongoClient.bulkWrite returns an error if auto-encryption is configured` in crud.prose.test.ts would have failed when it was implemented if we'd had this change at the time).

Reply:
This needs a reformat. can you push up the lint fixes?

Reply:
Yeah, fixed.